apps/src/lib/levelbuilder/script-editor/ScriptEditor.jsx

```diff
@@ -260,6 +262,18 @@ class ScriptEditor extends React.Component {
       is_migrated: this.props.isMigrated
     };
 
+    if (this.state.weeklyInstructionalMinutes) {
+      dataToSave.weekly_instructional_minutes =
```

Comment:
You should not need the null here because if the condition fails we just won't include anything at all for weekly_instructional_minutes